dashboard/src/components/DeploymentFormBody/AdvancedDeploymentForm.tsx

```diff
@@ -23,6 +23,9 @@ class AdvancedDeploymentForm extends React.Component<IAdvancedDeploymentForm> {
           value={this.props.appValues}
           className="editor"
         />
+        <p>
+          <b>Note:</b> Only comments from the original chart <code>values</code> will be preserved.
```

Comment:
```suggestion
          <b>Note:</b> Only comments from the original chart values will be preserved.
```

I suggest this change unless `values` refers to an actual eponymous file name, variable or similar.

Reply:
Yeah, perhaps even `values.yaml` (technically the chart `values` doesn't have comments, it's the source file which does).

BTW, @SimonAlling the extent of your vocabulary and attention to grammar of the English language astounds me :)

Reply:
`values.yaml` is at least as good!

Also, that comment of yours made my morning, @absoludity; thank you! :blush: